src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,9 @@
+package duke;
+
+public class DukeException extends Exception {
+
+    public DukeException(String message) {
+        super(message);
+    }
+
+}
```

Comment:
I feel that you can specify different exceptions to handle different cases of errors, such as InvalidCommandExceptions, InvalidDeadlineExceptions, and InvalidDescriptionExceptions.